gradle/wrapper.gradle

```diff
@@ -25,8 +25,8 @@ def wrapperUpdateTask = { name, label ->
                 if (version.empty) {
                     throw new GradleException("Cannot update wrapper to '${label}' version as there is currently no version of that label")
                 }
-                println "updating wrapper to $label version: $version.version (downloadUrl: $version.downloadUrl)"
-                distributionUrl version.downloadUrl
+                distributionUrl version.downloadUrl.replace("-bin", "-all")
```

Comment:
❌ I'd make it more precise with something like `.downloadUrl.replaceFirst(/-bin\.zip$/, '-all.zip')`

Reply:
How about we use `version.version` instead of `version.downloadUrl` and set `distributionType` on the wrapper task? So we wouldn't need to mangle with the URL at all?

Reply:
@wolfs if that works +1

Reply:
Good call @wolfs!
It works™

Reply:
\o/